sdk/identity/Azure.Identity/src/TokenCredentialOptions.cs

```diff
@@ -7,14 +7,12 @@
 namespace Azure.Identity
 {
     /// <summary>
-    /// Options to configure requests made to the OAUTH identity service
+    /// Options to configure requests made to the OAUTH identity service.
     /// </summary>
     public class TokenCredentialOptions : ClientOptions
     {
-        private static readonly Uri s_defaultAuthorityHost = new Uri("https://login.microsoftonline.com/");
-
         /// <summary>
-        /// The host of the Azure Active Directory authority.   The default is https://login.microsoftonline.com/
+        /// The host of the Azure Active Directory authority. The default is https://login.microsoftonline.com/. For common authority hosts for known azure cloud instances see <see cref="KnownAuthorityHosts"/>.
```

Comment:
Here you put common, above you use 'well known'. Would be good to go with one or the other.